### PR TITLE
Mock styles like gh

### PR DESCRIPTION
**Pull request by spoonben**
Because why not

### CODE REVIEW

**Comment by crawford** [comment]
image

**Comment by crawford** [comment]
tailor disable commit description

File: assets/templates/status.hbs [hunk 1]
```diff
@@ -4,24 +4,50 @@
...
+                font-size: 18px;
+                margin: 0;
+                padding: 0;
+            }
+            h1 {
+                background-color: #24292e;
+                color: rgba(255,255,255,0.75);
+                font-size: 30px;
+                margin: 0 auto;
+                max-width: 980px;
```
Comment (arithx) [inline assets/templates/status.hbs]: This looks better without the `max-width` on my screen

Comment (arithx) [reply assets/templates/status.hbs]: Nevermind, when I used inspection I used id instead of class for the header definition

**Pull request merged**